pkg/controller/core/resourceflavor_controller.go

```diff
@@ -128,7 +129,8 @@ func (r *ResourceFlavorReconciler) Delete(e event.DeleteEvent) bool {
 	log.V(2).Info("ResourceFlavor delete event")
 
 	if cqNames := r.cache.DeleteResourceFlavor(flv); len(cqNames) > 0 {
-		r.qManager.QueueInadmissibleWorkloads(cqNames)
+		ctx := ctrl.LoggerInto(context.Background(), log)
```

Comment:
in clusterqueue_controller.go you are just passing the context without a log, which I think it's fine as we are not logging anything in the function below.

For consistency, we can keep it simple and pass the bare context.

Reply:
ah, there are actually two instances in clusterqueue controller, one with a log and one without. I will change them all to use a bare context.